api/v1alpha4/azuremanagedcontrolplane_types.go

```diff
@@ -119,8 +118,8 @@ type AADProfile struct {
 	AdminGroupObjectIDs []string `json:"adminGroupObjectIDs"`
 }
 
-// SKU - AKS SKU.
-type SKU struct {
+// AksSKU - AKS SKU.
+type AksSKU struct {
```

Comment:
is it just me or the casing looks weird here?

Based on https://github.com/golang/go/wiki/CodeReviewComments#initialisms, seems like we should use `AKSSKU` or `AKSSku` ?

Reply:
Slight preference for AKSSku, so replaced with that